x-pack/plugins/cases/server/common/utils.test.ts

```diff
@@ -33,13 +33,20 @@ import {
   getCaseViewPath,
   isSOError,
   countUserAttachments,
+  isPersistableStateOrExternalReference,
 } from './utils';
 import { newCase } from '../routes/api/__mocks__/request_responses';
 import { CASE_VIEW_PAGE_TABS } from '../../common/types';
 import { mockCases, mockCaseComments } from '../mocks';
 import { createAlertAttachment, createUserAttachment } from '../services/attachments/test_utils';
 import type { CaseConnector } from '../../common/types/domain';
 import { ConnectorTypes } from '../../common/types/domain';
+import {
+  createAlertRequests,
+  createExternalReferenceRequests,
+  createPersistableStateRequests,
+  createUserRequests,
```

Comment:
nit: There are the same mocks in `x-pack/plugins/cases/server/mocks.ts` or `x-pack/test/cases_api_integration/common/lib/mock.ts`. Otherwise, we can extract these mock functions to `x-pack/plugins/cases/server/mocks.ts`.